<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oduncverme</class>
 <widget class="QDialog" name="oduncverme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Ödünç Alınan Kitaplar Listesi</string>
   </property>
  </widget>
  <widget class="QTableView" name="tw_oduncAlinan">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>256</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_oduncVer">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Ödüncü Ver</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Teslim Edilen Kitaplar Listesi</string>
   </property>
  </widget>
  <widget class="QTableView" name="tw_teslimEdilen">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>150</y>
     <width>256</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>135</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="le_uyeNo">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_kitapNo">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>60</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Uye No:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Kitap No:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>151</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Ödünç Verme Tarihi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit">
      <property name="date">
       <date>
        <year>2023</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>ÖDÜNÇ TESLİM ETME İŞLEMLERİ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
